### PR TITLE
US1591104 Loc Fix - Replace and add examples

### PR DESCRIPTION
**Pull request by v-thpra**
Made changes specified by [User Story 1591104](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1591104) 

Replaced one code example with another, added an example, and fixed a typo in the example.

### CODE REVIEW

**Comment by v-thpra** [comment]
#sign-off

**Comment by Rick-Anderson** [comment]
@v-thpra  check out the [Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/mvc/views/working-with-forms?view=aspnetcore-3.0&branch=pr-en-us-14311#no-selection)

At this point you'll probably need to use range. You can't add snippet tags to the source without doing that for every snippet.

File: aspnetcore/mvc/views/working-with-forms.md [hunk 1]
```diff
@@ -634,12 +634,14 @@ The *Views/Shared/EditorTemplates/CountryViewModel.cshtml* template:
 
 Adding HTML [\<option>](https://www.w3.org/wiki/HTML/Elements/option) elements isn't limited to the *No selection* case. For example, the following view and action method will generate HTML similar to the code above:
 
-[!code-csharp[](working-with-forms/sample/final/Controllers/HomeController.cs?range=114-119)]
+[!code-csharp[](working-with-forms/sample/final/Controllers/HomeController.cs?name=snippetNone)]
```
Comment (Rick-Anderson) [inline aspnetcore/mvc/views/working-with-forms.md]: It's looking for `snippetNone`. Is that what you intended?

**Comment by v-thpra** [comment]
@Rick-Anderson Thanks for flagging that! The "suggested change" I was working from had it wrong.

**Comment by guardrex** [comment]
@v-thpra :wave:

**Pull request merged**